src/scenes/Authentication/Register/RegisterForm.tsx

```diff
@@ -66,26 +68,38 @@ export const RegisterForm = ({ className, ...props }: RegisterFormProps) => {
         </div>
         <Form
           {...props}
-          validate={passwordMatching}
+          validate={validateFields}
           decorators={decorators}
           mutators={{ markConfirmPasswordTouched }}
         >
           {({ handleSubmit }) => (
             <form onSubmit={handleSubmit}>
               <SubmitError className={classes.formError} />
               <TextField
-                name="firstName"
+                name="realFirstName"
                 label="First Name"
                 placeholder="Enter First Name"
                 required
                 autoFocus
               />
               <TextField
-                name="lastName"
+                name="realLastName"
                 label="Last Name"
                 placeholder="Enter Last Name"
                 required
               />
+              <TextField
+                name="displayFirstName"
+                label="Display First Name"
```

Comment:
Let's call it _Public_ instead of _Display_, just like create/update user does.

Reply:
Changed